### ISSUE TITLE
Projectile: Fix Properties of a Correct Solution

### ISSUE THREAD

**Issue by smiths** [JacquesCarette/Drasil#3213]
Table 5 should be moved into the Properties of a Correct Solution section, since pland and doffset are outputs.

**Comment by smiths** [comment]
@harmanpreet-sagar and @ManvendraJani  I'm going through the Projectile issues and assigning them alternatively to each of you.  They shouldn't be difficult issues to close and getting Projectile as solid as possible will help with the other work that will be building on it.

**Comment by harmanpreet-sagar** [comment]
There is already a table in the **Properties of Correct Solution** which has `p_land` and `d_offset` listed as output constraints.

![image](https://github.com/JacquesCarette/Drasil/assets/110783820/4af1ce59-87eb-4985-83ef-a04632b76999)

The first table that uses `p_land` and `d_offset` after the **Table of Symbols** is:
![image](https://github.com/JacquesCarette/Drasil/assets/110783820/84dec9ea-03eb-411a-9649-f7eb38fd4175)

Is this the table that you are referring to?

**Comment by smiths** [comment]
The table is in the correct location now.  I checked both the [pdf](https://jacquescarette.github.io/Drasil/examples/projectile/SRS/srs/Projectile_SRS.pdf) and the [html](https://jacquescarette.github.io/Drasil/examples/projectile/SRS/srs/Projectile_SRS.html#Sec:CorSolProps) versions.  I was referring to the pdf version previously.  If I remember correctly Table 5 was appearing outside of the Properties of a Correct Solution section.  My guess is that the updates we did to the LaTeX compilation process fixed the problem.  :smile:

Regardless we can close this issue!  Good work @harmanpreet-sagar.  :smile:
